include/z64math.h

```diff
@@ -94,15 +94,14 @@ typedef struct {
 #define IS_ZERO(f) (fabsf(f) < 0.008f)
 
 // Trig macros
-#define DEGF_TO_BINANG(degreesf) (s16)(degreesf * 182.04167f + .5f)
-#define RADF_TO_BINANG(radf) (s16)(radf * (32768.0f / M_PI))
-#define RADF_TO_DEGF(radf) (radf * (180.0f / M_PI))
-#define DEGF_TO_RADF(degf) (degf * (M_PI / 180.0f))
-#define BINANG_ROT180(angle) ((s16)(angle - 0x7FFF))
-#define BINANG_SUB(a, b) ((s16)(a - b))
+#define DEGF_TO_BINANG(degreesf) (s16)((degreesf) * 182.04167f + .5f)
+#define RADF_TO_BINANG(radf) (s16)((radf) * (0x8000 / M_PI))
+#define RADF_TO_DEGF(radf) ((radf) * (180.0f / M_PI))
+#define BINANG_ROT180(angle) ((s16)((angle) - 0x7FFF))
+#define BINANG_SUB(a, b) ((s16)((a) - (b)))
 #define DEG_TO_RAD(degrees) ((degrees) * (M_PI / 180.0f))
```

Comment:
Shouldn't it be `DEG_TO_RADF`? Or am I just confused about the reasoning behind the F in these macros?

And I guess this question also applies to `BINANG_TO_RAD`

Reply:
That's a good question. I have no idea what the idea with the "F" is. Probably "float" but I feel like that's redundant with degrees/radians *always* being floats.

As you can see it's currently not consistent either to use that F or not. and `binang` does not use "S" either
In other spots `F`/`S` is used to distinguish radians/binangle (e.g. Math_CosF/CosS)

So I would move towards renaming the macros to not use the "F"

Reply:
i would say remove as well. I guess it was intended to help people remember to put `.0f` on the numbers? but yea its implied imo

Reply:
I'm guessing BINANG is implied to be shorts, which is why the "S" is not included for those, and yeah I guess the F could be omitted from the RAD/DEG stuff as well if we follow the same logic